django_project/base/models/location_site.py

```diff
@@ -0,0 +1,26 @@
+# coding=utf-8
+"""Site model definition.
+
+"""
+
+from django.contrib.gis.db import models
+from base.models.location_type import LocationType
+
+
+class LocationSite(models.Model):
+    """Location Site model."""
+
+    location_type = models.ForeignKey(
+        LocationType,
+        models.SET_NULL,
+        null=True,
```

Comment:
This should always have a type

Reply:
Also we need to add validation on the save that this cannot be changed. (If we have created a point type site we cannot change it to a line type site... (we'd have to create a new one)